Bottom: 11a95c9592045318cd25a0fa4631ccbb0ea056fc
Top:    546230af519e3539f1e7f13a89e7b1d050a64a76
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 16:53:33 +0000

find.c: Remove unnecessary PIN/VIA test from SetThing()

This is not needed since commit d581a90edc7fb7c0b4c6351611cc4a6689d7620b
which put similar code into ADD_PV_TO_LIST(), the only caller which
could have previously resulted in a SetThing call with the wrong type.


---

diff --git a/src/find.c b/src/find.c
index 0f85f3e..d10499d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -405,11 +405,6 @@ SetThing (int type, void *ptr1, void *ptr2, void *ptr3)
   thing_ptr2 = ptr2;
   thing_ptr3 = ptr3;
   thing_type = type;
-  if (type == PIN_TYPE && ptr1 == NULL)
-    {
-      thing_ptr1 = ptr3;
-      thing_type = VIA_TYPE;
-    }
   return true;
 }
